dist/datasource.ts

```diff
@@ -73,6 +73,7 @@ export default class GoogleStackdriverDatasource {
             for (let point of series.points) {
               datapoints.push([point.value[valueKey], Date.parse(point.interval.endTime).valueOf()]);
             }
+            datapoints.reverse();
```

Comment:
Why does this need to be reversed? Does stackdriver give it back in the wrong order?

Reply:
Stackdriver API returns timeseries in "reverse chronological order" and supports pagination. That breaks grafana's datapoint picker.